src/layouts/SideNavLayout.tsx

```diff
@@ -162,6 +174,17 @@ const SideNav = () => {
                       onClick={() => logOut(client, true)}
                     />
                   </Logout>
+                  <Version>
```

Comment:
So, since it's an endpoint, what happens when there is an error and we have no link and no version ?
(Same regarding the loading)

Reply:
It has been handled this was
- Loading have receive a skeleton, in place of the expected text to be dispayed
- If an error occur, the version section is hidden and the global error catch will display the error toast